params/computation_cost_params.go

```diff
@@ -46,7 +51,9 @@ const (
 	EqComputationCost             = 220
 	IszeroComputationCost         = 165
 	AndComputationCost            = 288
+	XorComputationCost            = 657
 	OrComputationCost             = 160
+	NotComputationCost            = 1289
```

Comment:
Please add somewhere in this file
```go
OpcodeComputationCostLimit = 100000000
OpcodeComputationCostLimitCancun = 120000000
OpcodeComputationCostLimitInfinite = uint64max?
OpcodeComputationCostLimitOverride = 0  // from node flag
```

Reply:
Aggregated the parameters in computation_cost_params.go https://github.com/klaytn/klaytn/commit/3709fd8a22b92ef8553e5d422e216d8fc0436e99.